### ISSUE TITLE
Default.tsx case sensitivity

### ISSUE THREAD

**Issue by chaseadam** [ItsRiprod/DeskThing#4]
possibly related to a "case-insensitive" filesystem.

On Fedora 40, had to change several references of "Default" to "default" related to "Default.tsx"

        modified:   DeskThing/src/views/default/default.tsx
        modified:   DeskThing/src/views/views.tsx

```
diff --git a/DeskThing/src/views/default/default.tsx b/DeskThing/src/views/default/default.tsx
index aed1200..1c2f9a0 100644
--- a/DeskThing/src/views/default/default.tsx
+++ b/DeskThing/src/views/default/default.tsx
@@ -1,5 +1,5 @@
 import { IconDevice } from '../../components/todothingUIcomponents';
-import './Default.css';
+import './default.css';
 import React from 'react';

 const Default: React.FC = (): JSX.Element => {
diff --git a/DeskThing/src/views/views.tsx b/DeskThing/src/views/views.tsx
index e2ddf69..151b990 100644
--- a/DeskThing/src/views/views.tsx
+++ b/DeskThing/src/views/views.tsx
@@ -1,7 +1,7 @@
 import { useState, useEffect, Suspense, lazy  } from 'react';
 import ButtonHelper, { Button, EventFlavour } from '../helpers/ButtonHelper';
 import socket from '../helpers/WebSocketService';
-import Default from './Default/Default';
+import Default from './default/default';
 import './views.css';

 import Overlay from '../components/Overlay/Overlay';
```

**Comment by ItsRiprod** [comment]
Which release is this for? This was resolved in the most recent version of the App (prerelease v0.5.0-alpha I believe)

default.tsx was removed altogether and replaced by dashboard.tsx and all of the folders are now lowercase

There is an error when overwriting an existing repo where Github will not change the capitalization of the files - leading to issues similar to this

**Comment by ItsRiprod** [comment]
As this has been resolved in v0.5.0+ I'm going to mark this as resolved

If anyone is still having this issue - clear your workspace and do a fresh install so the capitalization on the files is correct (or do so manually)
